disnake/ext/commands/slash_core.py

```diff
@@ -332,10 +332,12 @@ def parents(
 
     @property
     def description(self) -> str:
+        """:class:`str`: The slash sub command's description. Shorthand for :attr:`.Option.description`"""
```

Comment:
```suggestion
        """:class:`str`: The slash sub command's description. Shorthand for :attr:`self.body.description <Option.description>`."""
```
not sure if `Option.description` is easy to understand in this context, so being more explicit like this would be good